ramls/order.raml

```diff
@@ -60,6 +60,13 @@ resourceTypes:
           Update a purchase order with given {id}
           - if request does not include po_lines or includes "po_lines": [] or "po_lines": null - update just purchase order summary
           - if request includes array of "po_lines" - update PO lines as per request
+        queryParameters:
+          deleteHoldingAndItems:
```

Comment:
We should rename this param to 'deleteHoldings' to make it consistent with receiving

Reply:
done